lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -254,6 +305,70 @@ describe('BasicLtiLaunchApp', () => {
     });
   });
 
+  describe('multiple fetches', () => {
```

Comment:
```suggestion
  describe('concurrent fetching of groups and content URL', () => {
```

I think a slightly more descriptive name here will make these tests easier to follow.